tests/test_spam_mixin.py

```diff
@@ -0,0 +1,137 @@
+from __future__ import absolute_import
+from datetime import datetime
+import mock
+from nose.tools import *  # noqa PEP8 asserts
+from modularodm.exceptions import ValidationValueError, ValidationTypeError
+
+from framework.auth import Auth
+
+from tests.base import OsfTestCase
+from tests.factories import UserFactory, CommentFactory
+
+from website.project.model import SpamMixin
+
+
+class TestSpamMixin(OsfTestCase):
+
+    def setUp(self):
+        super(TestSpamMixin, self).setUp()
+        self.comment = CommentFactory()
+        self.auth = Auth(user=self.comment.user)
+
+    @mock.patch('website.project.model.SpamMixin.flag_spam')
```

Comment:
Because this is a unit test and the `test_flag_spam` tests that unit. In one case I do have to make it more of an integration test.